packages/webpack/loader.js

```diff
@@ -50,7 +50,9 @@ module.exports = async function(source) {
             out.push(`export var ${ident} = ${JSON.stringify(exported[ident])};`);
         });
 
-        out.push(`export var styles = ${JSON.stringify(result.details.result.css)};`);
+        if(options.styleExport !== false) {
```

Comment:
Can this be added to options as a default with a value of `true`, so this check could then be `if(!options.styleExport)`?

Reply:
I though adding `styleExport : true` to [`processor.js`](https://github.com/tivac/modular-css/blob/master/packages/processor/processor.js#L46) would set the default, but not seeing the expected result.

Reply:
`styleExport` isn't something the processor knows anything about, that's a bundler-specific thing that should probably go into [`plugin.js`](https://github.com/tivac/modular-css/blob/39153d76186e567eaaf3f8e57e2c9beb90d8aa0b/packages/webpack/plugin.js#L19-L22).

Something like this, probably:

```js
 var options = Object.assign(
    Object.create(null),
    { styleExport : true },
    args
);
```

(or broken out into `const defaults = { ... }` or something & then mixed into the `options`)

Reply:
Aha, ok. Thanks.

Reply:
Still working on this, maybe.